package.json

```diff
@@ -80,7 +80,7 @@
     "prepublish": "npm run build"
   },
   "engines": {
-    "node": "0.10.x",
-    "npm": "3.x"
+    "node": ">=0.10.x",
+    "npm": ">=3.x"
```

Comment:
Last I checked using npm with `>=` and `.x` might be invalid. Need to `.0` or leave out to patch/minor version.